src/GeoPing.FrontEnd/src/routers/routesComponent.tsx

```diff
@@ -47,15 +47,13 @@ export default class Routes extends React.Component<IRoutesComponentProps, any>
             <Route exact={true} path="/checklist" component={ChecklistPage}/>
             <Route exact={true} path="/checkin" component={CheckinPage}/>
             {
-              this.props.roleUser === ERoleUser.Admin ? (
+              this.props.roleUser === ERoleUser.Admin && (
```

Comment:
what will happen if not admin try to go to this route via direct link? (without passing through pages)

Reply:
it will throw on the page about